packages/api/src/command/medical/document/document-query.ts

```diff
@@ -73,6 +74,12 @@ export async function queryDocumentsAcrossHIEs({
     cxDocumentRequestMetadata,
   });
 
+  await appendDocQueryProgressWithSource({
+    patient: updatedPatient,
+    requestId,
+    reset: true,
+  });
+
   getDocumentsFromCW({
```

Comment:
Downstream of `getDocumentsFromCW()` we also initialize the patient's DQ status, on `initPatientDocQuery()`.

Not only it would be good to have a single doc query initialization, but also it might be that the CW initialization code messes up with this one.

I'd consider removing the CW one and having only one DQ init code. But it needs to be tested, so it might be good to move this to its own PR.